tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -92,157 +257,91 @@ describe('amo/components/AddonReviewList', () => {
+      const root = render({
         addon: {
           ...fakeAddon,
           icon_url: 'http://foo.com/hax.png',
         },
       });
-      const img = root.querySelector('.AddonReviewList-header-icon img');
-      expect(img.src).toEqual(fallbackIcon);
+      const img = root.find('.AddonReviewList-header-icon img');
+      expect(img.prop('src')).toEqual(fallbackIcon);
     });
 
     it('renders a hidden h1 for SEO', () => {
-      const root = renderToDOM({ addon: fakeAddon });
-      const h1 = root.querySelector('.AddonReviewList-header h1');
-      expect(h1.className).toEqual('visually-hidden');
-      expect(h1.textContent).toEqual(`Reviews for ${fakeAddon.name}`);
+      const root = render({ addon: fakeAddon });
+      const h1 = root.find('.AddonReviewList-header h1');
+      expect(h1).toHaveClassName('visually-hidden');
+      expect(h1.text()).toEqual(`Reviews for ${fakeAddon.name}`);
     });
 
     it('produces an addon URL', () => {
-      const root = findRenderedComponentWithType(
-        render(), AddonReviewListBase);
-      expect(root.addonURL()).toEqual(`/addon/${fakeAddon.slug}/`);
+      expect(render().instance().addonURL())
+        .toEqual(`/addon/${fakeAddon.slug}/`);
     });
 
     it('produces a URL to itself', () => {
-      const root = findRenderedComponentWithType(
-        render(), AddonReviewListBase);
-      expect(root.url()).toEqual(`/addon/${fakeAddon.slug}/reviews/`);
+      expect(render().instance().url())
+        .toEqual(`/addon/${fakeAddon.slug}/reviews/`);
     });
 
     it('requires an addon prop to produce a URL', () => {
-      const root = findRenderedComponentWithType(render({
-        addon: null,
-      }), AddonReviewListBase);
-      expect(() => root.addonURL()).toThrowError(/cannot access addonURL/);
+      expect(() => render({ addon: null }).instance().addonURL())
+        .toThrowError(/cannot access addonURL/);
     });
 
     it('configures a paginator with the right URL', () => {
-      const tree = render();
-      const root = findRenderedComponentWithType(tree, AddonReviewListBase);
-      const paginator = findRenderedComponentWithType(tree, Paginate);
-
-      expect(paginator.props.pathname).toEqual(root.url());
+      const root = render();
+      expect(root.find(Paginate).prop('pathname'))
+        .toEqual(root.instance().url());
     });
 
     it('configures a paginator with the right Link', () => {
-      const paginator = findRenderedComponentWithType(render(), Paginate);
-      expect(paginator.props.LinkComponent).toEqual(Link);
+      expect(render().find(Paginate).prop('LinkComponent')).toEqual(Link);
     });
 
     it('configures a paginator with the right review count', () => {
-      const paginator = findRenderedComponentWithType(
-        render({ reviewCount: 500 }), Paginate);
-      expect(paginator.props.count).toEqual(500);
+      const root = render({ reviewCount: 500 });
+      expect(root.find(Paginate).prop('count')).toEqual(500);
```

Comment:
`toHaveProp` matcher here too. I'm just commenting on them all so you can easily tell they're all fixed in the diff/PR comment view. 😅